src/components/activity-list/ActivityListEmptyState.tsx

```diff
@@ -16,13 +16,19 @@ const Container = styled(Column)({
   width: 200,
 });
 
-const ActivityListEmptyState = ({ children, emoji, label }) => {
+type ActivityListEmptyStateProps = {
+  children?: React.ReactNode;
+  emoji: string;
+  label: string;
+};
+
+const ActivityListEmptyState = ({ children, emoji, label }: ActivityListEmptyStateProps) => {
   const { top: topInset } = useSafeAreaInsets();
   const { colors, isDarkMode } = useTheme();
 
   return (
     <View>
-      {children}
```

Comment:
What does doubling children here do?

Reply:
Might not be needed, but it's an optional property so it's checking truthiness before rendering.

Reply:
I don't think it's needed